### PR TITLE
Replace ts-node with tsx, needed for sd v4

### PR DESCRIPTION
**Pull request by lukasoppermann**
## List of notable changes:

-

## What should reviewers focus on?

-

## Steps to test:

<!--
Help reviewers test the feature by providing steps to reproduce the behavior.

1. Open the preview documentation that has been deployed in this pull request
2. Go to # page
3. Verify that # behaves as described in the pull request description

1.
1.
1.

## Supporting resources (related issues, external links, etc):

-

## Contributor checklist:

- [ ] All new and existing CI checks pass
- [ ] Tests prove that the feature works and covers both happy and unhappy paths
- [ ] Any drop in coverage, breaking changes or regressions have been documented above
- [ ] All developer debugging and non-functional logging has been removed
- [ ] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change
- [ ] Verify the design tokens changed in this PR are expected using the diffing results in this PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "name": "@primer/primitives",
   "version": "8.2.3",
   "description": "Typography, spacing, and color primitives for Primer design system",
+  "engines": {
+    "node": ">=18.18.0 <18.19.0"
```
Comment (Eric-Guo) [inline package.json:6]: May I know the reason why can not support node v20?

**Comment by Eric-Guo** [comment]
I try to down grade node 18 and found I can only get 18.20.4, so this requirement still can not meet.

**Comment by Eric-Guo** [comment]
I can following [SO](https://stackoverflow.com/a/77350307/262826) to set `ignore-engines true` in .yarnrc to ignore this engine setting now.

**Comment by lukasoppermann** [comment]
@Eric-Guo there is an issue with a dependency, I think it is eslint, which requires higher node version.

**Comment by thomasbrus** [comment]
would appreciate that, also ran into this issue

**Comment by lukasoppermann** [comment]
Will be fixed in here: https://github.com/primer/primitives/pull/1038

**Comment by thomasbrus** [comment]
Thanks 🙌

**Pull request merged**